dart/packages/fury/lib/src/codegen/exception/fury_codegen_exception.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:fury/src/exception/fury_exception.dart';
+
```

Comment:
Could we move all exceptions into one file, or merge related exception classes into one file? 

We don't take java as reference